app/main.py

```diff
@@ -2,0 +1,23 @@
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    command_name = parsed_command[0]
+    source_file = parsed_command[1]
+    destination_file = parsed_command[2]
+
+    src_file_path = os.path.abspath(source_file)
+    dst_file_path = os.path.abspath(destination_file)
```

Comment:
Code Style: Use `os.path.join` to concatenate parts of the path. In that case, your app will be cross-platform and will use either `/` or `\` depending on OS.